src/api_calls/orders_calls.js

```diff
@@ -76,3 +76,28 @@ export function DeleteOrder(id){
 
 }
 
+export function UpdateOrder(id,patchRequestBody){
```

Comment:
You can rename this function to `updateOrder`.

Reply:
I-am dat remane in commint-ul acesta: 
Small fixes to order calls